admin/templates/login/login-link-sent.html

```diff
@@ -1,21 +1,51 @@
 {% extends 'base.html' %}
+{% import "govuk-design-system-templates/error-summary.html" as error_summary %}
+{% import "govuk-design-system-templates/text-input.html" as textInput %}
 
 {% block title %}Check your email (login){% endblock %}
 
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
 
+            {{ error_summary.renderFor(form) }}
+
             <h1 class="govuk-heading-l">
                 Check your email
             </h1>
 
             <p class="govuk-body">
-                A login link has been sent to your email address:
+                A security code has been sent to your email address:
                 <strong>{{ email_address }}</strong>.
-                Please click on this link to login
+                Please enter the security code
+            </p>
+
+            <p class="govuk-body govuk-!-margin-bottom-8">
+                If it's still not there, we can
+                <a href="{{ url_for('admin.sign_in_with_security_code') }}?resend=true" class="govuk-link govuk-link--no-visited-state">resend the email</a>.
             </p>
 
+            <form method="POST">
+                {{ form.csrf_token }}
```

Comment:
Is this reference to csrf_token still needed?

Reply:
Yeah it is. Needs to be referenced in all flask forms on the app otherwise Flask will complain that it's missing